pkgs/development/interpreters/bats/default.nix

```diff
@@ -22,6 +22,9 @@ stdenv.mkDerivation rec {
 
   inherit doCheck;
   checkPhase = ''
+    # TODO: reenable this test when fixed
+    sed -i '/test works even if PATH is reset/a skip' test/bats.bats
+
```

Comment:
Can you expand any on how we'll know? I don't see a recent issue/PR at bats-core that looked clearly like the same issue.

I'll take a look locally as well.